src/enforce/runners/run/index.js

```diff
@@ -1,7 +1,22 @@
 // @flow
+/**
+ * A function which returns whether a combination of
+ * rule + value is true or false
+ *
+ * @param {any} value - the value being tested
+ * @param {string} key the name of the rule being run
+ * @param {string} tests an object containing the group of tests in the current run
+ * @param {string} rules an object containing all the rules available for the current pass
+ * @return {boolean} value validation result
+ */
```

Comment:
which plugin are you using to generate those kind of description before your functions?

Reply:
@ealush

Reply:
It's called [JSDoc](http://usejsdoc.org/). Supported by github and most code editors.